javascript/rexuws/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "rexuws": "https://github.com/Aderinom/rexuws-update-deps"
```

Comment:
a fork ?

Reply:
Yes. Updated the version of uws to allow newer node version to run it. 
Didn't really think about the ergononics at that point though.

Already read your comment. Makes sense to block it until then.